api/src/main/resources/Reference_Application_Order_Entry_and_Allergies_Concepts-20.xml

```diff
@@ -1221,7 +1221,7 @@
 <concept_reference_source concept_source_id="21" name="CIEL" description="Columbia International eHealth Laboratory concept ID" creator="1" date_created="2013-07-03 15:55:03.0" retired="false" retired_by="1" uuid="21ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
 <concept_reference_source concept_source_id="24" name="IMO ProblemIT" description="Intelligent Medical Objects, Inc. problem/disease/finding datasource" creator="1" date_created="2013-10-01 13:34:15.0" retired="false" retired_by="1" uuid="24ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
 <concept_reference_source concept_source_id="27" name="NDF-RT NUI" description="Unique numeric indicator from NDF-RT drug information database" creator="1" date_created="2014-02-07 22:15:18.0" retired="false" retired_by="1" uuid="27ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
-<concept_reference_source concept_source_id="28" name="FDA Route of Administration" description="Description. This standard provides for all routes of administration for drugs. Source. COMIS Reference table (which is used by the Drug Product Reference File to generate Approved Drug Products with Therapeutic Equivalence Evaluations (a.k.a. The Orange Book)), and the Drug Registration and Listing Database, Office of Epidemiology and Biosurveillance Database. Also, the names (but not the definitions or the numeric codes) that are represented in blue have been harmonized with the E2B route of administration terms for the International Conference on Harmonization (ICH)." creator="1" date_created="2015-01-16 18:06:56.0" retired="false" retired_by="1" uuid="28ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
+<concept_reference_source concept_source_id="28" name="FDA Route of Administration" description="Description. This standard provides for all routes of administration for drugs. Source. COMIS Reference table (which is used by the Drug Product Reference File to generate Approved Drug Products with Therapeutic Equivalence Evaluations (a.k.a. "The Orange Book")), and the Drug Registration and Listing Database, Office of Epidemiology and Biosurveillance Database. Also, the names (but not the definitions or the numeric codes) that are represented in blue have been harmonized with the E2B route of administration terms for the International Conference on Harmonization (ICH)." creator="1" date_created="2015-01-16 18:06:56.0" retired="false" retired_by="1" uuid="28ADDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD" changed_by="1"/>
```

Comment:
Aren't the double quotes around `The Orange Book` the cause of the build failure?

Reply:
@dkayiwa yes they are. My thought I wasn't supposed to make the changes directly on the XML file since it was just a replica of content from somewhere. In my stupid thought I was looking for the original file to make changes on.